Modules/socketmodule.h

```diff
@@ -325,6 +325,9 @@ typedef struct {
     PyTime_t sock_timeout;     /* Operation timeout in seconds;
                                         0.0 means non-blocking */
     struct _socket_state *state;
+#ifdef MS_WINDOWS
+    int quickack;
```

Comment:
```suggestion
    unsigned int quickack: 1;
```

It could fit on 1-bit unsigned int.

Reply:
It could.  I kind of like the fact that the data type matches the one being given by / returned to Python though.  If you feel strongly about it I can make this change.

Reply:
Ah, yes it's being in Python afterwards. Not sure it will save much though (and possibly we won't save anything due to alignment). Let's forget about this.